### PR TITLE
fix fadeIn bug

### PR DESCRIPTION
**Pull request by MarkLeef**
*REPLACE ALL OF THIS TEXT WITH A DESCRIPTION OF YOUR CODE CHANGES*

Try to make sure your changes conform to the [Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/). If you have 40 minutes you'd like to have well-spent, watch [Modern Swift API Design](https://developer.apple.com/videos/play/wwdc2019/415/) from WWDC 2019.

### CODE REVIEW

File: Sources/AudioKit/Audio Files/AVAudioPCMBuffer+Processing.swift [hunk 1]
```diff
@@ -203,7 +203,7 @@ extension AVAudioPCMBuffer {
             fadeInPower = sampleTime / inTime
 
         } else {
-            fadeInPower = exp(log(10) * sampleTime / inTime)
+            fadeInPower = exp(log(100) * sampleTime / inTime)
```
Comment (aure) [inline Sources/AudioKit/Audio Files/AVAudioPCMBuffer+Processing.swift:206]: Why this change?

Comment (MarkLeef) [reply Sources/AudioKit/Audio Files/AVAudioPCMBuffer+Processing.swift:206]: I think I edited it by mistake

Comment (MarkLeef) [reply Sources/AudioKit/Audio Files/AVAudioPCMBuffer+Processing.swift:206]: ok
